management-common-v2/pom.xml

```diff
@@ -20,11 +20,11 @@
       <artifactId>management-core</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-servlet</artifactId>
-      <scope>provided</scope>
-    </dependency>
+<!--    <dependency>-->
```

Comment:
Why not just deleted (this and the half dozen other cases)?

Reply:
I meant to.